addons/io_hubs_addon/components/operators.py

```diff
@@ -640,7 +641,7 @@ def draw(self, context):
         layout.prop(self, "relative_path")
 
     def execute(self, context):
-        dirname = os.path.dirname(self.filepath)
+        #dirname = os.path.dirname(self.filepath) #dirname fails if path selected in the Blender File View is relative (starts with //)
```

Comment:
I think this line can be completely removed.  Also, dirname seems to handle relative paths fine on my system (Linux), so maybe it's OS dependant.

Reply:
> n windows an absolute path always starts with :.

Ah, yes, that's probably it.

> I didn't change anything to the behavior of the import of the other lines but the filepath.

You did change the behavior of the import because `self.filepath` isn't necessarily going to be the same as `self.directory + self.files[0].name`.  Try adding an image texture and selecting five files from the bottom up, you'll see the bottom file in the browser field, but the top file will be the one that actually goes into the image texture (unless Blender thinks you're trying to import a sequence, in which case the behavior may be different).  Now, it's true that this isn't quite 100% consistent behavior within Blender (but it's close), and the way Blender does it isn't as intuitive (but I think you're pretty safe in assuming that the top selected file, is the one that gets loaded into the field when opening multiple files at a time), so let's see what @keianhzo and @j-conrad think, but if the behavior is changed then the comment needs updating as well to reflect the new behavior and that it differs from Blender's default.

Re: the `def draw` function.
I don't remember exactly why I had this there initially, but it was probably because it was easy to add and would prevent mistakes if anyone needed to add in messages or more complex UI elements (plus you also don't have to worry about whether the properties are hidden or not).

> @Exairnous Yes, the line can be removed, I uploaded a 2. commit where I removed it.

Weird, the line is still there for me, just commented out.  Don't get me wrong, the explanation is useful to know, but I feel like it'd be more useful in a best practices document, rather than on a commented out line of code in a source file.  Actually, this line is used in other places, so they'll need updating as well.